### PR TITLE
Zombienet Equilibrium chain support

### PR DESCRIPTION
**Pull request by MucTepDayH16**
This PR adds Equilibrium/Genshiro session support

### CODE REVIEW

File: javascript/packages/orchestrator/src/paras.ts [hunk 1]
```diff
@@ -88,6 +88,11 @@ export async function generateParachainFiles(
     if (plainData.relay_chain) plainData.relay_chain = relayChainSpec.id;
     if (plainData.genesis.runtime?.parachainInfo?.parachainId)
       plainData.genesis.runtime.parachainInfo.parachainId = parachain.id;
+    if (
```
Comment (pepoviola) [inline javascript/packages/orchestrator/src/paras.ts:91]: Hi @MucTepDayH16 can we move this logic to `clearAuthorities` in `equilibrium.ts` and decorate that function to remove this custom logic here.

Thanks!!

**Comment by pepoviola** [comment]
Hi @MucTepDayH16, thanks for your contribution!! small comment inline and the is good to merge 👍 

Thanks!!

File: javascript/packages/orchestrator/src/paras.ts (hunk 1 above)
Comment (MucTepDayH16) [reply javascript/packages/orchestrator/src/paras.ts]: Ok, fixing it

**Review by pepoviola — APPROVED** [review]

**Comment by pepoviola** [comment]
Looks great, thanks!! Can you share  a link to the official image to run in our ci?
Thanks!!

**Comment by MucTepDayH16** [comment]
> Looks great, thanks!! Can you share a link to the official image to run in our ci? Thanks!!

Great, thank you too for merging. Our node image: https://hub.docker.com/r/equilab/eq-para.

**Pull request merged**